src/setupProxy.js

```diff
@@ -0,0 +1,25 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+// NOTE: Set your proxy with .env (use .env.sample)
+// otherwise it will fallback to localserver:8000
+
+// backround link on manual proxy config:
+// https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually
+
+var backendserver = process.env.REACT_APP_BACKEND;
+if (
+  process.env.REACT_APP_BACKEND == null ||
```

Comment:
Would it work with `=== null`? Not used to see `==` anymore.

Reply:
lets be strict on comparing null - fine with me